test/src/test_lu_mult.cpp

```diff
@@ -0,0 +1,66 @@
+/// @file test_lu_mult.cpp
+/// @brief Test LU multiplication
+//
+// Copyright (c) 2022, University of Colorado Denver. All rights reserved.
+//
+// This file is part of <T>LAPACK.
+// <T>LAPACK is free software: you can redistribute it and/or modify it under
+// the terms of the BSD 3-Clause license. See the accompanying LICENSE file.
+
+#include <catch2/catch.hpp>
+#include <plugins/tlapack_stdvector.hpp>
+#include <tlapack.hpp>
+#include <testutils.hpp>
+#include <testdefinitions.hpp>
+
+using namespace tlapack;
+
+TEMPLATE_LIST_TEST_CASE("lu multiplication is backward stable", "[lu check][lu][qrt]", types_to_test)
+{
+    srand(1);
+
+    using matrix_t = TestType;
+    using T = type_t<matrix_t>;
+    using idx_t = size_type<matrix_t>;
+    typedef real_type<T> real_t;
+
+    idx_t n;
+
+    n = GENERATE(1, 2, 6, 9, 20, 23, 28);
```

Comment:
28 is a bit large for the unit tests, unless you have a good reason to test large matrices, i would limit it to 20. Remember that we are eventually testing this for thousands of routines for multiple different types and orientations and parameters. That can add up.